src/bagit/__init__.py

```diff
@@ -1481,10 +1481,7 @@ def _make_parser():
 
     checksum_args = parser.add_argument_group(
         _("Checksum Algorithms"),
-        _(
```

Comment:
This was a formatting change that my local ruff suggested. I'm not sure why CI didn't care too...